client/src/commonMain/kotlin/com/algolia/search/model/response/ResponseListIndices.kt

```diff
@@ -45,21 +45,30 @@ public data class ResponseListIndices(
         /**
          * Last build time in seconds.
          */
-        @SerialName(Key.LastBuildTimeS) val lastBuildTimeS: Int,
+        @SerialName(Key.LastBuildTimeS) val lastBuildTimeSOrNull: Int? = null,
         /**
          * Number of pending indexing operations.
          */
-        @SerialName(Key.NumberOfPendingTasks) val numberOfPendingTasks: Int,
+        @SerialName(Key.NumberOfPendingTasks) val numberOfPendingTasksOrNull: Int? = null,
         /**
          * A boolean which says whether the index has pending tasks.
          */
-        @SerialName(Key.PendingTask) val pendingTask: Boolean,
+        @SerialName(Key.PendingTask) val pendingTaskOrNull: Boolean? = null,
         @SerialName(Key.Replicas) val replicasOrNull: List<IndexName>? = null,
         @SerialName(Key.Primary) val primaryOrNull: IndexName? = null,
         @SerialName(Key.SourceABTest) val sourceABTestOrNull: IndexName? = null,
         @SerialName(Key.ABTest) val abTestOrNull: ResponseABTestShort? = null
     ) {
 
+        public val lastBuildTimeS: Int
+            get() = lastBuildTimeSOrNull!!
```

Comment:
nitpick: I know this is the current pattern to use `!!` , however, it is much better to use [requireNotNull](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require-not-null.html) function instead.

Reply:
Noted! I'll use it next time, I prefer to follow best practices indeed